### PR TITLE
Add an api to access the user link sites. 

### PR DESCRIPTION
**Pull request by Anas-hameed**
## Description: 
The API add a API endpoint to access user linked site from LMS to panel. 

## JIRA:
https://edlyio.atlassian.net/browse/EDLY-5608?atlOrigin=eyJpIjoiMDBkNzY3NTQwN2U3NDhlMjgzYjAzMTQ1Y2Q5NGY0YTQiLCJwIjoiaiJ9

### CODE REVIEW

File: openedx/features/edly/api/serializers.py [hunk 1]
```diff
@@ -49,3 +50,30 @@ def get_mobile_enabled(self, obj):  # pylint: disable=unused-argument
...
+    Serializer for Mutisite access endpoint
+    """
+    sub_org = EdlySubOrgSerializer(read_only=True)
+
+    class Meta:
+        """
+        Meta attribute for the MutiSiteAccess Model
+        """
+        model = EdlyMultiSiteAccess
+        fields = ['id', 'sub_org']
```
Comment (taimoor-ahmed-1) [inline openedx/features/edly/api/serializers.py:79]: we can simplify this into one single serializer as edly_sub_org is already present in edly_multisite_access model we can directly fetch those field values

File: openedx/features/edly/api/v1/views/user_mutisites.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import urllib.parse 
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from openedx.features.edly.api.serializers import MutiSiteAccessSerializer
+from openedx.features.edly.models import EdlyMultiSiteAccess
+
+
+class MutiSitesViewSet(viewsets.ViewSet):
```
Comment (taimoor-ahmed-1) [inline openedx/features/edly/api/v1/views/user_mutisites.py:13]: type `class MultisitesViewset`

File: openedx/features/edly/api/v1/views/user_mutisites.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        If the request is successful, the request returns an HTTP 200 "OK" response.
+
+        The HTTP 200 response has the following values.
+
+        * list of sub-organization linked with your email
+
+    """
+    permission_classes = ()
+    authentication_classes = ()
```
[the remark this answers is not in the document]
Comment (Anas-hameed) [reply openedx/features/edly/api/v1/views/user_mutisites.py:33]: As from CORS configurations, panel backend is allowed to make request to LMS. We need to get sites information of the user, that are linked to email to sign in by giving an additional popup incase of mutisites user for selecting site. 

We can also make the request using the token for panel backend user.

File: openedx/features/edly/api/serializers.py (hunk 1 above)
Comment (Anas-hameed) [reply openedx/features/edly/api/serializers.py:79]: resolved

File: openedx/features/edly/api/v1/views/user_mutisites.py (hunk 2 above)
Comment (Anas-hameed) [reply openedx/features/edly/api/v1/views/user_mutisites.py:13]: Fixed

**Review by daniyalfarman — APPROVED** [review]

**Review by muhammadali124 — APPROVED** [review]
LGTM

**Pull request merged**